test/web_console/railtie_test.rb

```diff
@@ -45,7 +46,26 @@ class RailtieTest < ActiveSupport::TestCase
         app.config.web_console.mount_point = '/customized/path'
         app.initialize!
 
-        assert_equal Middleware.mount_point, '/customized/path'
+        assert_equal '/customized/path', Middleware.mount_point
+      end
+    end
+
+    test 'config.mount_point supports relative_url_root' do
+      new_uninitialized_app do |app|
+        app.config.relative_url_root = '/relative/path'
+        app.initialize!
+
+        assert_equal '/relative/path/__web_console', Middleware.mount_point
+      end
+    end
+
+    test 'config.mount_point is prior than config.relative_url_root' do
+      new_uninitialized_app do |app|
+        app.config.web_console.mount_point = '/customized/path'
+        app.config.relative_url_root = '/relative/path'
+        app.initialize!
+
+        assert_equal '/customized/path', Middleware.mount_point
```

Comment:
Indeed :-) I was thinking about using the `Middleware.mount_point` as an absolute path, but it might not be natural as its name. I will update it.